src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -219,9 +219,12 @@ def __init__(self, run: Run, stream=False, run_mode=RunMode.Test):
 
         self._dump_meta_file()
         if run.flow:
-            flow_obj = load_flow(source=run.flow)
-            # TODO(2898455): refine here, check if there's cases where dag.yaml not exist
-            self._eager_mode = isinstance(flow_obj, EagerFlow)
+            try:
```

Comment:
Can we do such catch in the main list function?
![image](https://github.com/microsoft/promptflow/assets/47586720/619f0bf0-9ec9-4c2a-ba44-6b662439dd75)

Reply:
The error happens in CLI side, not in SDK side. I think SDK side already handled this.

Reply:
Offline synced. I'll add another try catch in CLI's list to guard.